src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/bytecode/patch/SponsorBlockBytecodePatch.kt

```diff
@@ -41,11 +40,13 @@ import org.jf.dexlib2.iface.reference.StringReference
 @Patch
 @DependsOn(
     dependencies = [
-        VideoInformationPatch::class, // updates video information and adds method to seek in video
+        IntegrationsPatch::class,
         VideoIdPatch::class,
-        PlayerControlsBytecodePatch::class,
+        // uses video seek to skip segments, and playback speed to schedule upcoming skips
+        VideoInformationPatch::class,
+        // to check for and ignore Shorts (SponsorBlock does not support Shorts)
         PlayerTypeHookPatch::class,
```

Comment:
```suggestion
        // Required to skip segments on time.
        VideoInformationPatch::class,
        // Used to prevent SponsorBlock from running on Shorts because SponsorBlock does not yet support Shorts.
        PlayerTypeHookPatch::class,
```